_newsletters/2019-05-01-may-2019.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: May 2019 Newsletter
+category: newsletters
+last_modified_at: 
+tags: newsletters
+newsletter-year: 2019
+sidenav: about_us
+---
+
+## Summary
+
+We released a new search parameter, our legacy results page goes away June 28, &amp; soon users will be deactivated after 90 days of inactivity.
+
+## New Feature
+
+### Sitelimit Parameter Now Available on Our Main Index
+
+We recently added support for a parameter called sitelimit to our main web index. Sitelimit is a way to narrow down a search, similar to Collections, allowing flexibility for you and your searchers. This is a nice way to offer alternative search scopes to your searchers via radio buttons or checkboxes, essentially creating on-the-fly collections.
+
+### How Does it work?
+
+The sitelimit parameter overrides the <a href="https://search.gov/manual/domains.html">Domains list</a>. However, to work, the paths included have to be within the scope of the Domains list in the admin center for the site.
+
+Simply add the following to the URL for the search page:
+
+`&sitelimit=www.exampledomain.gov/limittothisfolder`
+
+Multiple paths can be combined with a +, for example:
+
+`&sitelimit=www.exampledomain.gov/limittothisfolder+folderarchive.exampledomain.gov`.
+
+### How to add it
+
+Sitelimit parameters can be added via an input in the search box
+
+```<input type="hidden" name="sitelimit" id="sitelimit" value="www.exampledomain.gov/limittothisfolder">```
```

Comment:
do we need to change these code block backtick characters to their own lines? looks like the rest of the code is confused, and still blue.